### DISCUSSION TITLE
Feature Request: Time-series Bar Chart stack series within group by

### DISCUSSION THREAD

**Discussion by HeathLee** [apache/superset#20716] [General]
![image](https://user-images.githubusercontent.com/7382922/179146599-f4539551-db31-4d32-855e-9603d45ad059.png)
If using multi metrics and group by at the same time, and want stack metrics, all metrics in different groups were stacked together,
but I want stack with splited group like this
![image](https://user-images.githubusercontent.com/7382922/179147292-f6e75c8a-3545-4d4b-87b6-38c3c96cb271.png)

**Comment by kamalkeshavani-aiinside** [comment]
This is really required!
